cmake/setup.sh.in

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
 
 # get path to script
-export FCCSWBASEDIR="$( cd "$(dirname "$0")" ; pwd -P )"
-export FCC_DETECTORS=$FWCCSWBASEDIR
+export FCCSWBASEDIR=@CMAKE_INSTALL_PREFIX@
+export FCC_DETECTORS=$FCCSWBASEDIR
 export FCC_PYTHIACARDS=$FCCSWBASEDIR
 
 export PATH=$PATH:$FCCSWBASEDIR/scripts:$FCCSWBASEDIR/bin
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$FCCSWBASEDIR/lib
 export PYTHONPATH=$PYTHONPATH:$FCCSWBASEDIR/lib:$FCCSWBASEDIR/python
 export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:$FCCSWBASEDIR
+
+
+# make sure gaudi scripts and modules are available
+export PYTHONPATH=$PYTHONPATH:$FCCVIEW/lib:$FCCVIEW/python
+export PATH=$PATH:$FCCVIEW/scripts:$FCCVIEW/bin
```

Comment:
I agree with you, but for a first step we should make sure the cvmfs installation are working (and for other users who set up their own gaudi (like me :) ) this will probably have no effect as it just appends `/lib` etc to the paths, which won't do anything). But yes, the externals should set up these paths for Gaudi to work properly. Since I cannot setup FCCSW without the Gaudi python modules in the PYTHONPATH, I propose to leave it like this now and delete these lines once the setup script for the fcc view is fixed.

Reply:
In that case I would rather fix the template in the fcc view and patch the current setup.sh installed in cvmfs. I can do that so if you want.

Reply:
Ah great, if you can do that, that would the best solution. Let me know when the patch is applied, then I'll test it again.

Reply:
I already applied it, can you please give it a try?

Reply:
Yes, it's working now, thanks